workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -129,6 +129,10 @@ public abstract class StatefulWorkflow<
     context: RenderContext
   ): RenderingT
 
+  override val identifier: WorkflowIdentifier by lazy {
```

Comment:
Since we're thread-confined while looking these up, would it make sense to use `LazyThreadSafetyMode.NONE` to avoid the locking behavior?